src/libguac/client.c

```diff
@@ -701,6 +739,122 @@ int guac_client_owner_supports_required(guac_client* client) {
     
 }
 
+/**
+ * A callback function that is invokved by guac_client_owner_notify_join() to
+ * notify the owner of a connection that another user has joined the
+ * connection, returning zero if the message is sent successfully, or non-zero
+ * if an error occurs.
+ *
+ * @param user
+ *     The user to send the notification to, which will be the owner of the
+ *     connection.
+ *
+ * @param data
+ *     The data provided to the callback, which is the user that is joining the
+ *     connection.
+ *
+ * @return
+ *     Zero if the message is sent successfully to the owner, otherwise
+ *     non-zero, cast as a void*.
+ */
+static void* guac_client_owner_notify_join_callback(guac_user* user, void* data) {
+
+    const guac_user* joiner = (const guac_user *) data;
+
+    if (user == NULL)
+        return (void*) ((intptr_t) -1);
+
+    char* logOwner = "owner";
+    if (user->info.name != NULL)
+        logOwner = (char *) user->info.name;
+
+    char* logJoiner = "anonymous";
+    char* sendJoiner = "";
+    if (joiner->info.name != NULL) {
+        logJoiner = (char *) joiner->info.name;
+        sendJoiner = (char *) joiner->info.name;
+    }
```

Comment:
These should be `lowercase_with_underscores` rather than `headlessCamelCase`.

Reply:
Got it - should be fixed.